test/page-object.test.ts

```diff
@@ -296,17 +295,16 @@ test(
   })
 );
 
-describe('calculateIosTopOffset', () => {
```

Comment:
I do not understand why this test lived in a `page-object` test file. Extracted separately